.editorconfig

```diff
@@ -0,0 +1,10 @@
+root = true
+
+[*]
+charset = utf-8
+end_of_line = lf
+insert_final_newline = true
+
+[*.php]
+indent_style = tab
+indent_size = 8
```

Comment:
I am not sure if there is any preference for this. Do we need to set it at all or could we let all tools display tabs as wide as they "normally" do?